sensor_msgs/Image img
BoundingBboxes bboxes
---
int32 res
